gui.pl

```diff
@@ -0,0 +1,115 @@
+% Display the board's current state
+display_board(BoardSize, BoardState):-
+    display_top_border(BoardSize),
+    nl,
+    display_current_board(1, BoardSize, BoardState),
+    nl,
+    display_bottom_border(BoardSize).
+
+% Display the board's top border: including the top indices (A, B...) and the separator line
+display_top_border(BoardSize):-
+    SpaceLimit is floor(BoardSize / 2) + 7,
+    print_char_repeat(' ', 1, SpaceLimit),
+    IndexLimit is ceil(BoardSize / 2),
+    print_index_row('A', 1, IndexLimit),
+    nl,
+    SlashLimit is SpaceLimit - 1,
+    print_char_repeat(' ', 1, SlashLimit),
+    print_pointers_row(BoardSize),
+    nl,
+    DunderLimit is SlashLimit - 2,
```

Comment:
Why DunderLimit? (what's dunder?) and not SeperatorLimit?

Reply:
Done :)